src/components/UI/DrawerItems/index.tsx

```diff
@@ -1,30 +1,138 @@
-import React from 'react';
-import ChevronLeftIcon from '@mui/icons-material/ChevronLeft';
-import { Box, IconButton, Divider } from '@mui/material';
+import * as React from 'react';
+import AppBar from '@mui/material/AppBar';
+import Box from '@mui/material/Box';
+import CssBaseline from '@mui/material/CssBaseline';
+import Divider from '@mui/material/Divider';
+import Drawer from '@mui/material/Drawer';
+import IconButton from '@mui/material/IconButton';
+import List from '@mui/material/List';
+import ListItem from '@mui/material/ListItem';
+import ListItemButton from '@mui/material/ListItemButton';
+import ListItemIcon from '@mui/material/ListItemIcon';
+import ListItemText from '@mui/material/ListItemText';
+import MenuIcon from '@mui/icons-material/Menu';
+import Toolbar from '@mui/material/Toolbar';
+import Typography from '@mui/material/Typography';
+import MovieIcon from '@mui/icons-material/Movie';
+import BookIcon from '@mui/icons-material/Book';
+import StarRateIcon from '@mui/icons-material/StarRate';
 
-import './styles/index.scss';
+const drawerWidth = 240;
 
 interface Props {
+	window?: () => Window;
 	toggleDrawer: () => void;
 }
 
-const DrawerItems = ({ toggleDrawer }: Props) => {
+export default function DrawerItems(props: Props) {
+	const { window } = props;
+	const [mobileOpen, setMobileOpen] = React.useState(false);
+
+	const handleDrawerToggle = () => {
+		setMobileOpen(!mobileOpen);
+	};
+
+	const drawer = (
+		<div>
+			<Toolbar />
+			<Divider />
+			<List>
+				{['Movies', 'Manga', 'Anime', 'Top Rated'].map((text, index) => (
+					<ListItem key={text} disablePadding>
+						<ListItemButton>
+							<ListItemIcon>
+								{
+									[
+										<MovieIcon key="movies" />,
+										<BookIcon key="manga" />,
+										<MovieIcon key="anime" />,
+										<StarRateIcon key="top-rated" />,
+									][index]
+								}
+							</ListItemIcon>
+							<ListItemText primary={text} />
+						</ListItemButton>
+					</ListItem>
+				))}
+			</List>
+		</div>
+	);
+
+	const container =
+		window !== undefined ? () => window().document.body : undefined;
+
 	return (
-		<>
-			<Box sx={{ width: 300 }}>
-				<div className="drawer-header">
-					<IconButton onClick={toggleDrawer}>
-						<ChevronLeftIcon color="primary" />
+		<Box sx={{ display: 'flex' }}>
+			<CssBaseline />
+			<AppBar position="fixed">
```

Comment:
This component drawerItems was originally supposed to contain just the different List items that you create. From the looks of this it seems you are trying to place a appbar in the drawer and create nested drawers?

It would be much easier to understand the direction you are going if you follow the template so reviewers can have context.